package/debian/control

```diff
@@ -0,0 +1,15 @@
+Source: kiele
+Section: devel
+Priority: optional
+Maintainer: Everett Hildenbrandt <everett.hildenbrandt@runtimeverification.com>
+Build-Depends: clang-8 , cmake , kframework , libcrypto++-dev , libjemalloc-dev , libprocps-dev , libprotobuf-dev , libsecp256k1-dev , libssl-dev , opam , pkg-config , protobuf-compiler
+Standards-Version: 3.9.6
+Homepage: https://github.com/runtimeverification/iele-semantics
+
+Package: kiele
+Architecture: any
+Section: devel
+Priority: optional
+Depends: libcrypto++-dev , libjemalloc-dev , libmpfr-dev , libprotobuf-dev , libsecp256k1-dev
```

Comment:
this doesn't look right. If you're not compiling C code, which I don't believe the user of the IELE package is, then you ought not to need the development version of these libraries.